### PR TITLE
Update @apollo/experimental-nextjs-app-support to v0.8.0-OLD

### PR DESCRIPTION
**Pull request by theodesp**
## Tasks

- [ ] I have signed a [Contributor License Agreement (CLA)](https://github.com/wpengine/faustjs#contributor-license-agreement) with WP Engine.
- [ ] If a code change, I have written testing instructions that the whole team & outside contributors can understand.
- [ ] I have written and included a comprehensive [changeset](https://github.com/wpengine/faustjs/blob/canary/DEVELOPMENT.md#deployment) to properly document the changes I've made.

## Description

Updates @apollo/experimental-nextjs-app-support to v0.8.0 in both the package and the example projects

## Testing

1. Cleanup all your node_modules from the `examples/next/app-router`.
2. Run `npm i` and `npm run build`
3. Start the app-router example project. It should works as. expected.

### CODE REVIEW

File: packages/experimental-app-router/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
   },
   "peerDependencies": {
     "@apollo/client": ">=3.8.0",
-    "@apollo/experimental-nextjs-app-support": ">=0.5.0",
```
Comment (blakewilson) [inline packages/experimental-app-router/package.json:51]: I'm not sure if we should update the peer dependency's version here, that would be a breaking change.

https://github.com/semver/semver/issues/502#issuecomment-466501843

By upgrading the example project and the dev dependency we are still avoiding the dependabot vulnerability but giving users the choice to use 0.5.0 through 0.8.0.

Comment (theodesp) [reply packages/experimental-app-router/package.json:51]: Its a v1.0 change so the risk is justified. Also we cannot allow the user to have a vulnerable version anyway

Comment (blakewilson) [reply packages/experimental-app-router/package.json:51]: Agreed. We should still increment `experimental-app-router` with a `major` bump though.

**Comment by TeresaGobble** [comment]
@theodesp  I tried updating to the fix in my own PR to pass the `Audit Dependencies` check, but it then causes a cascade of failing checks elsewhere, similar to what you have here. I'm not able to pinpoint why- just wanted to confirm that I'm seeing failures after the update attempt as well.

**Pull request closed without merging**